code/localization/localize.cpp

```diff
@@ -62,7 +63,7 @@ bool *Lcl_unexpected_tstring_check = nullptr;
 // NOTE: with map storage of XSTR strings, the indexes no longer need to be contiguous,
 // but internal strings should still increment XSTR_SIZE to avoid collisions.
 // retail XSTR_SIZE = 1570
-// #define XSTR_SIZE	1675
+// #define XSTR_SIZE	1798
```

Comment:
After examining the provided `new_xstrs.txt` and #5627, the new strings should start at 1675 and not 1700. The logic here is that all built-in XSTR IDs should be contiguous. The new value of XSTR_SIZE should be 1774 and the new XSTRs should go from 1675 to 1773.

Reply:
You're not concerned about bumping into mods using this IDs for something? Also note that XSTR_SIZE define is just a comment now. I can still update the comment, but the define is technically gone.

Reply:
The logic here is actually related to collisions, but sort of reversed. The engine shouldn't have to avoid collisions with mods using XSTRs, mods should be avoiding collisions with the engine. The principle is similar to port allocations for TCP/UDP where lower ports (IDs) are reserved for specific uses and applications (mods) should avoid colliding with reserved regions.